src/containers/pages/JurisdictionAssignment/helpers/Slider/tests/slider.test.tsx

```diff
@@ -39,31 +67,111 @@ describe('JurisdictionAssignment/Slider', () => {
 
     wrapper.update();
 
-    // Akros_1 is selected by default during mount since it has a threshold of 80 > 0
+    // Akros_1 and Ona office are selected by default, since threshold of both > 0
     expect(wrapper.find('div.slider-section').text()).toMatchInlineSnapshot(`"Risk  0%00100"`);
-    expect(wrapper.find('div.info-section').text()).toMatchInlineSnapshot(
-      `"NUMBER OF STRUCTURES IN SELECTED JURISDICTIONS159"`
-    );
+    expect(wrapper.find('div.info-section').length).toEqual(2);
+    expect(
+      wrapper
+        .find('div.info-section')
+        .first()
+        .text()
+    ).toMatchInlineSnapshot(`"NUMBER OF STRUCTURES IN SELECTED JURISDICTIONS169"`);
+    expect(
+      wrapper
+        .find('div.info-section')
+        .last()
+        .text()
+    ).toMatchInlineSnapshot(`"2 jurisdiction(s) selected"`);
 
     // simulate a change on the input slider. will set a thresh-hold value
-    // that will not select any of the jurisdictions
+    // that will deselect one of the jurisdictions
 
     /** we really should not be doing this, but I am currently unable to simulate
      * the change event
      */
     act(() => {
-      (wrapper.find('InputRange').props() as any).onChange(81);
+      (wrapper.find('InputRange').props() as any).onChange(71);
     });
 
-    // the risk value should be now different 2
-    expect(wrapper.find('.risk-label').text()).toMatchInlineSnapshot(`"81%"`);
+    // the risk value should be now different 71%
+    expect(wrapper.find('.risk-label').text()).toMatchInlineSnapshot(`"71%"`);
 
     // invoke the method that now actually causes the auto-selection to happen
+    (wrapper.find('InputRange').props() as any).onChangeComplete(71);
+
+    // the structure and jurisdiction counts should now reflect just Akros_1 being selected
+    expect(
+      wrapper
+        .find('div.info-section')
+        .first()
+        .text()
+    ).toMatchInlineSnapshot(`"NUMBER OF STRUCTURES IN SELECTED JURISDICTIONS159"`);
+    expect(
+      wrapper
+        .find('div.info-section')
+        .last()
+        .text()
+    ).toMatchInlineSnapshot(`"1 jurisdiction(s) selected"`);
+
+    // simulate another change to deselect all jurisdictions
+    /** we really should not be doing this, but I am currently unable to simulate
+     * the change event
+     */
+    act(() => {
+      (wrapper.find('InputRange').props() as any).onChange(81);
+    });
+    expect(wrapper.find('.risk-label').text()).toMatchInlineSnapshot(`"81%"`);
     (wrapper.find('InputRange').props() as any).onChangeComplete(81);
+    // we should now have two jurisdictions selected
```

Comment:
I think this should read `no selected jurisdictions`

Reply:
thanks for catching that @p-netm - now fixed